cocos/ui/UIButton.cpp

```diff
@@ -156,22 +156,21 @@ bool Button::createTitleRendererIfNull() {
     return false;
 }
 
-void Button::createTitleRenderer()
-{
-    _titleRenderer = Label::create();
-    _titleRenderer->setAnchorPoint(Vec2::ANCHOR_MIDDLE);
-    addProtectedChild(_titleRenderer, TITLE_RENDERER_Z, -1);
+void Button::createTitleRenderer() {
```

Comment:
It seems like those changes are for different PR. Please create separate branch for each PR.

Reply:
Was it not done here? https://github.com/cocos2d/cocos2d-x/pull/19089